src/constants/index.jsx

```diff
@@ -4,6 +4,9 @@ export const MODAL_SHOWN = 'MODAL_SHOWN'
 
 export const DATE_RANGE_MIN = '2011-12-01T12:00:00.000Z'
 
+export const DATE_VALIDATION_FORMAT = 'M/D/YYYY';
```

Comment:
You could add a comment above this line with a link to `https://day.js.org/docs/en/display/format` so it's clearer this is set by dayjs.